### ISSUE TITLE
Have you adopted sigmoid before the calculation of discriminator loss?

### ISSUE THREAD

**Issue by hitachinsk** [HannH/DMFN#6]
Hello, I'm reading the implementation of DMFN from your codes, which is a fabulous job, however when I viewed the calculation of GAN loss I was confused why there is no sigmoid imposed after the logits generated from discriminator?
